doc/rbd/iscsi-target-cli.rst

```diff
@@ -0,0 +1,167 @@
+=============================================================
+Configuring the iSCSI Target using the Command Line Interface
+=============================================================
+
+The Ceph iSCSI gateway is the iSCSI target node and also a Ceph client
+node. The Ceph iSCSI gateway can be a standalone node or be colocated on
+a Ceph Object Store Disk (OSD) node. Completing the following steps will
+install, and configure the Ceph iSCSI gateway for basic operation.
+
+**Requirements:**
+
+-  A running Red Hat Ceph Storage 2.3 cluster
+
+-  The following packages must be installed from a Red Hat Ceph Storage
+   repository:
+
```

Comment:
The existing docs freely intermix "Red Hat Enterprise Linux" and CentOS -- probably safe to keep it and possibly add "... / CentOS" wherever "Red Hat Enterprise Linux" is named.

I don't think those available as pre-packages anywhere -- I think the goal will be to have the ceph-iscsi-XYZ packages available on download.ceph.com like the other software. For the others, we might just need to provide a Fedora Copr or something similar. I'll ask Ken about that.